components/BottomNavigation/src/ColorThemer/MDCBottomNavigationBarColorThemer.m

```diff
@@ -20,9 +20,7 @@ @implementation MDCBottomNavigationBarColorThemer
 
 + (void)applyColorScheme:(id<MDCColorScheme>)colorScheme
     toBottomNavigationBar:(MDCBottomNavigationBar *)bottomNavigationBar {
-  bottomNavigationBar.selectedItemTintColor = colorScheme.primaryLightColor;
-  bottomNavigationBar.unselectedItemTintColor = colorScheme.primaryDarkColor;
-  bottomNavigationBar.backgroundColor = colorScheme.secondaryColor;
+  bottomNavigationBar.selectedItemTintColor = colorScheme.primaryColor;
```

Comment:
This looks like a change in existing behavior.  Is there an bug?  Are people relying on the existing behavior?

Reply:
rob showed this to me offline: if we applied the color scheme to the bottom nav bar it would do weird coloring. so yes there was is a bug. If you only specified the primary color your secondary color would be the same as the primary and this would result in primary on primary background, icon and text becoming invisible.

Reply:
There are a few problems here:
* All of those properties are optional, but there is no check being performed.
* If the color scheme does not support any of the properites, no theming gets performed at all.
* Other components (AppBar, BottomAppBar, NavigationBar, ButtonBar, Buttons, FlexibleHeader) use `primaryColor` to set the background color.  
   * HeaderStackView uses primary and primaryLight as background colors.
   * FeatureHighlight uses primary and primaryLight for the background colors.
* Other components use colors for foreground elements but not background.
   * TabBar sets the selected/unselected to light/dark, but not the background color. 
   * Slider uses primary and lightPrimary for the track. 
   * PageControl uses light/dark for the indicators. 
   * ProgressView use primary and lightPrimary for track/progress.

This is the only component where we are using the same scheme to set both foreground/accent and background colors.  Given how MDCBasicColorSchemes are automatically generated and how primary/secondary colors are generally used in our components, it seems better for the BottomNavigation to **either** color the background color or the item tints.